themes/notmyidea/templates/base.html

```diff
@@ -71,7 +71,9 @@
 
         <footer id="contentinfo" class="body">
             <address id="about" class="vcard body">
-                The Women in Computer Science Undergraduate Committee (WiCS ugrad) is a sub-committee of the David R. Cheriton School of Computer Science Women in Computer Science Committee. Check out their <a href="https://cs.uwaterloo.ca/wics">website</a>!
+                The Women in Computer Science Undergraduate Committee (WiCS ugrad) is a sub-committee of the David R. Cheriton School of Computer Science Women in Computer Science Committee. Check out their <a href="https://cs.uwaterloo.ca/wics">website</a>!<br>
+                The Waterloo Women in Computer Science Committee values inclusivity and aims to include women and nonbinary individuals in Computer Science that come from all backgrounds.
+                We acknowledge that we are on the traditional territory of ‎ the Neutral, Anishnawbe and Haudenosaunee peoples. The University of Waterloo is situated on the Haldimand Tract, the land promised to the Six Nations that includes six miles on each side of the Grand River.
```

Comment:
An example of a proper acknowledgement can be found [here](https://starcon.io/) or in the email archives.